letslambda.json

```diff
@@ -58,7 +145,12 @@
         "LambdaExecutionRole": {
             "Type": "AWS::IAM::Role",
             "Properties": {
-                "ManagedPolicyArns": [ {"Ref": "LetsLambdaManagedPolicy"} ],
+                "ManagedPolicyArns": [
+                    {"Ref": "LetsLambdaManagedPolicy"},
+                    {"Ref": "LetsLambdaS3WriteManagedPolicy"},
+                    {"Ref": "LetsLambdaS3ReadManagedPolicy"},
+                    {"Ref": "LetsLambdaKmsKeyManagedPolicy"}
```

Comment:
Since we save private keys, having a read-only policy allows users to create an ec2 instance role and reuse our pre-defined policies more easily. Typically:

- `LetsLambdaManagedPolicy` is for the lambda execution
- `LetsLambdaS3WriteManagedPolicy` is for lambda but also for anything that would require write access to the bucket (ie: something that would have to write the `yml` file
- `LetsLambdaS3ReadManagedPolicy` is for any AWS resources that would need to retrieve assets in the bucket, but without having the risk of deleting anything
- `LetsLambdaKmsKeyManagedPolicy` to manage access to KMS key access for encryption, but also if private keys have been encrypted with KMS, then this would be required by the relevant consumers (ie: EC2 instances via a role).